src/pages/_app.tsx

```diff
@@ -72,6 +72,8 @@ const loadLocaleData = (locale: AvailableLocale): Promise<any> => {
       return import('../i18n/locale/sr.json');
     case 'sv':
       return import('../i18n/locale/sv.json');
+    case 'uk':
+      return import('../i18n/locale/uk.json');
```

Comment:
Not sure why you removed Bulgarian and Finnish from the list?
And I think when removing the languages, you didn't notice that Hebrew (hi), Hindi (hi), Romanian (ro) are already available from Weblate after #3597 had been merged. Please add those 3 back.

Reply:
I have been busy, but I got all of this sorted out.